gamestonk_terminal/stocks/behavioural_analysis/ba_controller.py

```diff
@@ -104,6 +106,10 @@ def print_help():
         print("   queries       top related queries with this stock")
         print("   rise          top rising related queries with stock")
         print("SentimentInvestor:")
+        print("   popularsi     show most popular stocks on social media right now")
```

Comment:
`ba: error: argument cmd: invalid choice: 'popularsi' `  Not sure which needs to be changed

Reply:
Probably in choices  (although would popular make more sense since you already specify the SI source?)